theme-kit/lib/createContentDeliveryClient.ts

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import type { Category, Culture, Newsroom, PrezlyClient } from '@prezly/sdk';
+import { ApiError, NewsroomGallery, SortOrder, Stories, Story } from '@prezly/sdk';
+
+const DEFAULT_PAGE_SIZE = 20;
+
+interface Params {
+    formats?: Story.FormatVersion[];
+    pinning?: boolean;
+    pageSize?: number;
+}
+
+export function createContentDeliveryClient(
```

Comment:
This is a simplified reimplementation of `PrezlyAPI` from the `theme-kit` package. The goal is move it to the `theme-kit` repo and replace `PrezlyAPI` with simpler more compact APIs, so it feels natural next to Next.js helpers (naming-wise too)